### DISCUSSION TITLE
How to add blacklist to custom zebrafish archr genome

### DISCUSSION THREAD

**Discussion by rlawltn9718** [GreenleafLab/ArchR#1896] [Questions / Documentation] [answered]
Hello, 
I am avidly using archr on my zebrafish data.

I noticed that blacklists were included in the genomes supported by ArchR(hg19,hg38,mm9,mm10).
Currently, I have finished generating my custom zebrafish genome according to the ArchR tutorial.

I was wondering how to add a blacklist to my custom zebrafish ArchR genome. 
How can I do that?

![image](https://user-images.githubusercontent.com/120088929/227158068-0c67800a-6650-47a6-9925-f2d3f3237a46.png)
[ArchR-plotEmbedding-224c01efd8984-Date-2023-03-20_Time-11-22-39.log](https://github.com/GreenleafLab/ArchR/files/11057649/ArchR-plotEmbedding-224c01efd8984-Date-2023-03-20_Time-11-22-39.log)

I want to add the blacklist to my genome with the hopes of fixing my stripe-like feature plots (as in the photo above). My guess is
the gene scores were incorrectly calculated.
If adding the blacklist isn't the solution, what other possibilities could have generated this issue?
I have exactly followed the tutorial.

Thanks in advance!

**Comment by Zepeng-Mu** [comment, marked as answer]
This is related to ggplot2 version that has been discussed before.

**Reply by rcorces** [reply]
thanks @Zepeng-Mu 

@kimjisoo18 Please search previous issues and discussions thoroughly before posting. Answer is here https://github.com/GreenleafLab/ArchR/issues/1731
